external/storm-jdbc/src/main/java/org/apache/storm/jdbc/common/HikariCPConnectionProvider.java

```diff
@@ -40,6 +44,10 @@ public synchronized void prepare() {
             Properties properties = new Properties();
             properties.putAll(configMap);
             HikariConfig config = new HikariConfig(properties);
+            if(properties.containsKey("dataSource.url"))
+                LOG.info("DataSource Url: " + properties.getProperty("dataSource.url"));
+            else if (config.getJdbcUrl() != null)
+                LOG.info("JDBC Url: " + config.getJdbcUrl());
```

Comment:
@tkuwabara789 can you put the if-else block in curly braces? And also create a JIRA and update the title of this PR?